suites/scimap/scimap_phenotyping.xml

```diff
@@ -46,14 +46,14 @@
             <param name="anndata" value="tutorial_data.h5ad" ftype="h5ad" />
             <param name="manual_gates" value="manual_gates.csv" ftype="csv" />
             <param name="gating_workflow" value="phenotype_workflow.csv" ftype="csv" />
-            <output name="output" file="tutorial_data_pheno.h5ad" compare="sim_size" delta="10" />
+            <output name="output" file="tutorial_data_pheno.h5ad" compare="sim_size" delta="200000" />
```

Comment:
I doubt that this is a good test. There are advanced methods to test the content of a hdf5 file that you can use idead.

Reply:
wow, yes, the test machine supports subkey, `obs/phenotype`.

Reply:
Yes, I think so. https://docs.galaxyproject.org/en/latest/dev/schema.html#assertions-for-h5-data